helpdesk_mgmt_timesheet_time_control/models/helpdesk_ticket.py

```diff
@@ -21,9 +21,10 @@ def _relation_with_timesheet_line(self):
     def _compute_show_time_control(self):
         result = super()._compute_show_time_control()
         for ticket in self:
-            ticket.show_time_control = (
+            if not (
                 ticket.project_id.allow_timesheets and ticket.team_id.allow_timesheet
-            )
+            ):
+                ticket.show_time_control = False
```

Comment:
I guess you reverted more than you should.

Reply:
@Yajo nope, your suggestion to turn a selection option into a bool I didn't agree with either.

Reply:
:+1: Oops I thought it *was* a bool already :open_mouth: